### PR TITLE
impr(quotes): Added 3 English quotes (aarnav593)

### PR DESCRIPTION
**Pull request by aarnav593**
### Checks

- [x] Adding quotes?
  - [x] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [x] Adding a language or a theme?
  - [x] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [x] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38622,6 +38622,25 @@
...
       "id": 7038
+    },
+    {
+      "text": "It is difficult to get a man to understand something, when his salary depends upon him not understanding it.",
+      "source": "Upton Sinclair",
+      "length": 108,
+      "id": 7039
+    },
+    {
+      "text": "I just saw a sign that said 'speed limit enforced by aircraft.' I know that means they're just scanning for fast-moving cars, but imagine going 90 in a 65 and getting bombed by a stealth bomber.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38633]: Use " instead of '

Comment (Miodec) [reply frontend/static/quotes/english.json:38633]: Actually, just remove - not really liking this quote.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8544039228

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38622,6 +38622,31 @@
...
       "id": 7038
+    },
+    {
+      "text": "It is difficult to get a man to understand something, when his salary depends upon him not understanding it.",
+      "source": "Upton Sinclair",
+      "length": 108,
+      "id": 7039
+    },
+    {
+      "text": "Hmm... Let's see here... It says:\"You could defraud your insurance company.\" Who in the world deep fries their insurance company?",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38633]: Sorry, dont like this quote either.

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -29921,7 +29921,7 @@
     },
     {
       "text": "You get anxious about confronting somebody in your life. That anxiety cripples you and you start wondering why you're anxious. Now you're becoming anxious about being anxious. Oh, no! Doubly anxious! Now you're anxious about your anxiety, which is causing more anxiety. Quick, where's the whiskey?",
-      "source": "Mark Manson, THE SUBTLE ART OF NOT GIVING A F***",
+      "source": "Mark Manson",
```
[the remark this answers is not in the document]
Comment (Miodec) [reply frontend/static/quotes/english.json:29924]: I think its fine. I would revert this.

[the remark this answers is not in the document]
Comment (aarnav593) [reply frontend/static/quotes/english.json:29924]: Should I only capitalize the first letters of the words (all caps looks quite annoying).

[the remark this answers is not in the document]
Comment (Miodec) [reply frontend/static/quotes/english.json:29924]: Sure

**Review by Miodec — APPROVED** [review]

**Comment by Miodec** [comment]
Thanks

**Pull request merged**